db/seeds.rb

```diff
@@ -177,6 +177,19 @@ def create_case(full_name,
     total_days = rand(0..25).round
     total_hours = rand(0.0..10.0).round
 
+    child.approvals.each do |approval|
```

Comment:
Added this to the seeds so we sometimes get values on the child_approvals, should help the dashboard look a little closer to real life.